lint-requirements.txt

```diff
@@ -1,9 +1,2 @@
-black
-flake8==6.1.0
-flake8-docstrings
-flake8-copyright
```

Comment:
Does ruff check for copyright notices? I think we'd need to use `--preview` flag: https://docs.astral.sh/ruff/rules/missing-copyright-notice/

Reply:
It does when `preview` is enabled. Just somehow it keeps complaining about it so I disabled it for now. I am currently taking a look at the `CPY` rule and checking where goes wrong.